rpa-builder/6.2/modules/ROOT/pages/toolbox-network-rest-call.adoc

```diff
@@ -1,170 +1,247 @@
 
-
 = REST Call
 
-You can use the _REST Call Action Step_ to fetch data directly from a
+Use the REST Call Action Step to fetch data directly from a
 server and to collect the required information in a simple way.
 
-You define the end point as you wish and you can assign parameters or
-process variables to the call. This enables you to flexibly address any
-REST interface. The response in JSON format (or XML) can be used as a
+You define the endpoint according to your needs and you can assign parameters or
+process variables to the call, which enables you to flexibly address any
+REST interface. You can use the response in JSON format (or XML) as a
 source variable during the remainder of the process run.
 
-== REST Call via a proxy
-
-You can also make the REST Call via a proxy server. To do this, the REST
-Call Action Step automatically retrieves the necessary information from
-the Network Settings, provided you have made the appropriate settings
-there.
-
-The Network Settings can be found in the Settings Action Step, which is
-located in the Workflow Initialization block.
-
-////
-For more information about the settings options, see chapter 2.6.5
-Settings.
-////
-
-== Properties
+== REST Call via a Proxy
 
-*REST Call Settings*
+To make REST calls via a proxy server:
 
-*Base Url* Here, the end point entered in the wizard is displayed.
+image:configure-proxy-settings.png[75%, 75%, The Settings option inside the Workflow Initialization block and the Network Settings configuration window.]

```

Comment:
Regarding alt text, Gary told me that the text should be enclosed in quotes if there are spaces in it.